src/Exception/InvalidParameterException.php

```diff
@@ -15,9 +15,12 @@
 
 use Sonata\CoreBundle\Exception\InvalidParameterException as CoreBundleException;
 
-/**
- * @deprecated Use Sonata\CoreBundle\Exception\InvalidParameterException instead
- */
-class InvalidParameterException extends CoreBundleException
-{
+if (class_exists(CoreBundleException::class)) {
+    class InvalidParameterException extends CoreBundleException
```

Comment:
I really don't know how it works.
Can we use a possibly non-existing class ?

Or should we write
```suggestion
if (class_exists(Sonata\CoreBundle\Exception\InvalidParameterException::class)) {
    class InvalidParameterException extends Sonata\CoreBundle\Exception\InvalidParameterException
```

?

Reply:
https://3v4l.org/kJ4Lm